sims/verilator/Makefile

```diff
@@ -116,48 +179,70 @@ model_header_debug = $(model_dir_debug)/V$(VLOG_MODEL).h
 model_mk = $(model_dir)/V$(VLOG_MODEL).mk
 model_mk_debug = $(model_dir_debug)/V$(VLOG_MODEL).mk
 
-#########################################################################################
+#============================================================================
+# selectively enable verbose logging
+#============================================================================
+# see GNU Make static patterns
+$(sim_vsrcs_printf) : %.printf.v : %.v
+	$(base_dir)/bin/enable_printfs $< $@ "$(ENABLE_PRINTF_PATTERN)"
+
+#============================================================================
 # build makefile fragment that builds the verilator sim rules
-#########################################################################################
-$(model_mk): $(sim_vsrcs) $(sim_common_files) $(EXTRA_SIM_REQS)
+#============================================================================
+$(model_mk): $(sim_vsrcs_printf) $(sim_common_files)
 	rm -rf $(model_dir)
 	mkdir -p $(model_dir)
-	$(VERILATOR) $(VERILATOR_OPTS) -o $(sim) -Mdir $(model_dir) -CFLAGS "-include $(model_header)"
+	$(VERILATOR) $(VERILATOR_OPTS) $(EXTRA_SIM_SOURCES) -o $(sim) \
```

Comment:
Nit: Can this be on one line or further split into multiple. Ditto for the debug version

Reply:
yes, done